lib/utils/oclif-utils.ts

```diff
@@ -69,3 +69,28 @@ export function capitanoizeOclifUsage(
 		.replace(/(?<=\s)[A-Z]+(?=(\s|$))/g, (match) => `<${match}>`)
 		.toLowerCase();
 }
+
+export async function getCommandsFromManifest() {
+	const fs = await import('fs');
+	let manifest;
+	try {
+		manifest = JSON.parse(
+			await fs.promises.readFile('./oclif.manifest.json', 'utf8'),
```

Comment:
Ah, somehow one of my review comments (the reason for requesting changes!) got lost. Hang on, I am writing it again...

Reply:
I downloaded the standalone zip file from the draft release page and got:
```
$ ./balena-cli-12.25.5/balena version -a
Oclif manifest not found.

Additional information may be available with the `--debug` flag.
For help, visit our support forums: https://forums.balena.io
For bug reports or feature requests, see: https://github.com/balena-io/balena-cli/issues/
```
The solution might be to test for `process.pkg` and conditionally determine the path for `oclif.manifest.json`, along the lines of:
https://github.com/balena-io/balena-cli/blob/v12.25.4/patches/all/open+7.1.0.patch#L10-L12

The standalone executable tests may have passed because they run on the same folder as the regular tests (I guess), so `./oclif.manifest.json` happened to exist on the current working dir.

Reply:
Ah! Yeah got caught out there.